integration-tests/smoke/keeper_test.go

```diff
@@ -642,6 +645,65 @@ var _ = Describe("Keeper Suite @keeper", func() {
 			}, "1m", "1s").Should(Succeed())
 		}
 
+		if testToRun == HandleKeeperNodesGoingDown {
+			By("takes down half of the keeper nodes and watches upkeeps get performed slower")
+			// Record how much time it takes to have all the registered upkeeps perform 5 times each (the first time)
+			const numberOfTimesUpkeepsShouldPerform int64 = 5
```

Comment:
just thinking, do we get any value by setting it to 5 instead of 1? If not, it makes the test slower without much gain